draft-ietf-acme-acme.md

```diff
@@ -306,36 +306,25 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "kid" (except for new-reg and revoke-cert requests).
```

Comment:
Just deleting "jwk" leaves a hole.  Don't you want something like the following?

```
  * "jwk" (only for requests to new-reg and revoke-cert resources)
  * "kid" (for requests to all resources except new-reg and revoke-cert)
```

These requirements should be reflected with MUSTs in the next paragraph.